.gitignore

```diff
@@ -0,0 +1,145 @@
+# 401 JS
```

Comment:
your node modules got pushed to github for this lab too - please keep an eye on that because it's really hard to grade when there are 27,000 changes in a pull request.